js/products/addItem.js

```diff
@@ -0,0 +1,56 @@
+import { postItemData } from "../fetch.js";
+
+const modalAddItem = document.querySelector("#addItemModal");
+const modalForm = document.querySelector(".modal-form");
+const button = document.querySelector("#modal-button");
+const addItem = document.querySelector(".add-product");
+const inputTitle = document.querySelector("#title");
+const inputDescription = document.querySelector("#description");
+const inputPrice = document.querySelector("#price");
+const inputImageUrl = document.querySelector("#imageUrl");
+const close = document.querySelectorAll(".closeAddModal")[0];
+
+const openAddModal = () => {
+  modalAddItem.style.display = "block";
+  close.addEventListener("click", closeAddModal);
+};
+
+const closeAddModal = () => {
+  modalAddItem.style.display = "none";
+  close.removeEventListener("click", closeAddModal);
+  addItem.removeEventListener("click", closeAddModal);
```

Comment:
Are you sure that a `closeAddModal` function is registered to this element?  
I believe it's an anonymous arrow function. 😃